samples/cellular/modem_shell/src/rest/rest_shell.c

```diff
@@ -240,11 +239,13 @@ int rest_shell(const struct shell *shell, size_t argc, char **argv)
 		case REST_SHELL_OPT_PRINT_RESP_HEADERS:
 			print_headers_from_resp = true;
 			break;
+
 		case '?':
+			mosh_error("Unknown option. See usage:");
```

Comment:
I'm not getting this error neither with `rest --foobar` nor `rest foobar`.

Reply:
Removing:
```
	if (argc < 3) {
		show_usage = true;
		goto end;
	}
```